src/cascadia/TerminalConnection/AzureConnection.cpp

```diff
@@ -800,4 +873,13 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
             }
         }
     }
+
+    std::wstring AzureConnection::_strFormatHelper(const wchar_t* const format, int i, const wchar_t* name, const wchar_t* ID)

```

Comment:
You could make this more generic and put it in the types library (and use a varargs function so it can be used for any pattern format...)

Reply:
I'll look into that after I'm done with VT sequence stuff